src/proof/inputs-generator.ts

```diff
@@ -60,19 +61,30 @@ type InputContext = {
   circuitQueries: Query[];
 };
 
-const circuitValidator: {
-  [k in CircuitId]: { maxQueriesCount: number };
+const allOperations = Object.values(QueryOperators);
+const v2Operations = [
+  Operators.NOOP,
+  Operators.EQ,
+  Operators.LT,
+  Operators.GT,
+  Operators.IN,
+  Operators.NIN,
+  Operators.NE
+];
+
+export const circuitValidator: {
+  [k in CircuitId]: { maxQueriesCount: number; supportedOperations: Operators[] };
 } = {
-  [CircuitId.AtomicQueryMTPV2]: { maxQueriesCount: 1 },
-  [CircuitId.AtomicQueryMTPV2OnChain]: { maxQueriesCount: 1 },
-  [CircuitId.AtomicQuerySigV2]: { maxQueriesCount: 1 },
-  [CircuitId.AtomicQuerySigV2OnChain]: { maxQueriesCount: 1 },
-  [CircuitId.AtomicQueryV3]: { maxQueriesCount: 1 },
-  [CircuitId.AtomicQueryV3OnChain]: { maxQueriesCount: 1 },
-  [CircuitId.AuthV2]: { maxQueriesCount: 0 },
-  [CircuitId.StateTransition]: { maxQueriesCount: 0 },
-  [CircuitId.LinkedNullifier]: { maxQueriesCount: 1 },
-  [CircuitId.LinkedMultiQuery10]: { maxQueriesCount: 10 }
+  [CircuitId.AtomicQueryMTPV2]: { maxQueriesCount: 1, supportedOperations: v2Operations },
+  [CircuitId.AtomicQueryMTPV2OnChain]: { maxQueriesCount: 1, supportedOperations: v2Operations },
+  [CircuitId.AtomicQuerySigV2]: { maxQueriesCount: 1, supportedOperations: v2Operations },
+  [CircuitId.AtomicQuerySigV2OnChain]: { maxQueriesCount: 1, supportedOperations: v2Operations },
+  [CircuitId.AtomicQueryV3]: { maxQueriesCount: 1, supportedOperations: allOperations },
+  [CircuitId.AtomicQueryV3OnChain]: { maxQueriesCount: 1, supportedOperations: allOperations },
+  [CircuitId.AuthV2]: { maxQueriesCount: 0, supportedOperations: [] },
+  [CircuitId.StateTransition]: { maxQueriesCount: 0, supportedOperations: [] },
+  [CircuitId.LinkedNullifier]: { maxQueriesCount: 1, supportedOperations: [] },
+  [CircuitId.LinkedMultiQuery10]: { maxQueriesCount: 10, supportedOperations: [] }
```

Comment:
also [] all operators, remove LinkedNullifier also

Reply:
removed `linkedNullifier`, added all operators for `LinkedMultiQuery10 `